### PR TITLE
dragonfly-reverb: 0.9.1 -> 0.9.4

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by markuskowa** [comment]
@GrahamcOfBorg build dragonfly-reverb

File: pkgs/applications/audio/dragonfly-reverb/default.nix [hunk 1]
```diff
@@ -5,8 +5,8 @@ stdenv.mkDerivation rec {
 
   src = fetchgit {
```
Comment (markuskowa) [inline pkgs/applications/audio/dragonfly-reverb/default.nix]: Would you mind changing it to `fetchFromGitHub`?

Comment (magnetophon) [reply pkgs/applications/audio/dragonfly-reverb/default.nix]: This:
```
  src = fetchFromGitHub {
    owner = "michaelwillis";
    repo = "dragonfly-reverb";
    rev = "0.9.3";
    sha256 = "0a1zj58bgfg5gpzn50bywv3hzh8rfign7gazi2r6wrn9ky71xyb7";
  };
``` 
doesn't build:
```
make[1]: Leaving directory '/build/source/freeverb3'
make -C dpf/dgl
make[1]: *** dpf/dgl: No such file or directory.  Stop.
make: *** [Makefile:15: libs] Error 2
```

I create [an issue](https://github.com/michaelwillis/dragonfly-reverb/issues/29).

**Review by FRidh — APPROVED** [review]
Note 0.9.4 has been releases in the meanwhile.

File: pkgs/applications/audio/dragonfly-reverb/default.nix (hunk 1 above)
Comment (markuskowa) [reply pkgs/applications/audio/dragonfly-reverb/default.nix]: Oh I see. The sub folder even dpf points to another repository.

Comment (c0bw3b) [reply pkgs/applications/audio/dragonfly-reverb/default.nix]: You would need to set `fetchSubmodules` to true like:

```nix
  src = fetchFromGitHub {
    owner = "michaelwillis";
    repo = "dragonfly-reverb";
    rev = "0.9.3";
    sha256 = "<hash>";
    fetchSubmodules = true;
  };
```

SHA hash might need to be recomputed

**Comment by magnetophon** [comment]
@FRidh Good catch.

**Comment by xeji** [comment]
@GrahamcOfBorg build dragonfly-reverb

**Pull request merged**